<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PageOpen</class>
 <widget class="QDialog" name="PageOpen">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QMainWindow,QDialog{background-color:#bdc3c7;}</string>
  </property>
  <widget class="QPushButton" name="startStopButton">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>130</y>
     <width>100</width>
     <height>100</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton{
border-radius: 20px;
border:5px 11c4d4;
background-repeat:no-repeat;
background-color:#ADB4B9;
}
QPushButton::hover{
background:lightgray;
}</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>30</y>
     <width>120</width>
     <height>40</height>
    </rect>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:20pt;&quot;&gt;下料页面&lt;/span&gt;&lt;/p&gt;&lt;p&gt;&lt;span style=&quot; font-size:20pt;&quot;&gt;&lt;br/&gt;&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QPushButton" name="exitButton">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>130</y>
     <width>100</width>
     <height>100</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton{
border-radius: 20px;
border:5px 11c4d4;
background-repeat:no-repeat;
background-color:#ADB4B9;
}
QPushButton::hover{
background:lightgray;
}</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/new/icon/icon/cancel_1.png</normaloff>:/new/icon/icon/cancel_1.png</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>80</width>
     <height>80</height>
    </size>
   </property>
  </widget>
  <widget class="QLabel" name="startOpen_Label">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>240</y>
     <width>70</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>开始下料</string>
   </property>
  </widget>
  <widget class="QLabel" name="exit_Label">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>240</y>
     <width>71</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>取消</string>
   </property>
  </widget>
 </widget>
 <resources>
  <include location="res.qrc"/>
 </resources>
 <connections/>
</ui>
